src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = (props) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = props.person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`
+          : 'I am not married'
+      }
+      </p>
```

Comment:
Don't forget about code style
Also remove this to the variable
```suggestion
    <section className="Person">
      <h2 className="Person__name">{`My name is ${name}`}</h2>
      
      {age && (
        <p className="Person__age">{`I am ${age}`}</p>
      )}
      
      <p className="Person__partner">
        {isMarried
          ? `${partnerName} is my ${partner'}`
          : 'I am not married'
      }
      </p>
```